lib/rails/observers/railtie.rb

```diff
@@ -20,6 +18,7 @@ class Railtie < ::Rails::Railtie
       end
 
       config.after_initialize do |app|
+        require 'active_record/base'
```

Comment:
I was getting a very odd error about a circular dependency in loading a constant (particularly, the name of a model) until I put this in. This was the only way I could get it to go away. Would you like to see the backtrace?